lighthouse-cli/bin.ts

```diff
@@ -187,18 +187,21 @@ function launchChromeAndRun(addresses: Array<string>,
       log.log('Lighthouse CLI', 'Launching Chrome...');
       return launcher.run();
     })
-    .then(() => lighthouseRun(addresses, config))
+    .then(() => lighthouseRun(addresses, config, opts.lighthouseFlags))
     .then(() => launcher.kill());
 }
 
-function lighthouseRun(addresses: Array<string>, config: Object) {
+function lighthouseRun(addresses: Array<string>, config: Object, lighthouseFlags?: Object) {
+  // Enable a programatic consumer to pass custom flags otherwise default to CLI.
+  lighthouseFlags = lighthouseFlags || flags;
```

Comment:
naming nit: since we have multiple "flags" at play.

1. up top: ` const flags = cli;` ==> ` const cliFlags = cli;`
2. here: `const flags = lighthouseFlags || cliFlags`
3. then we're keeping `flags` consistent as it moves from `bin` into `lighthouse/index`.

seem reasonable?